src/java.base/unix/classes/sun/nio/fs/UnixFileSystem.java

```diff
@@ -888,6 +899,9 @@ void move(UnixPath source, UnixPath target, CopyOption... options)
             return;
         } catch (UnixException x) {
             if (x.errno() != EXDEV && x.errno() != EISDIR) {
+                if (x.errno() == ENOTDIR) {
+                    x.setError(ENOENT);
+                }
```

Comment:
This is also a rename so need to drop this change too.

Reply:
Reverted in 0908e75.